NuGet.Docs/Consume/Command-Line-Reference.md

```diff
@@ -710,6 +710,104 @@ Specify the API key to save and an optional URL to the server that provided the
 
     nuget setapikey 4003d786-cc37-4004-bfdf-c4f3e8ef9b3a -Source http://example.com/nugetfeed
 
+##  Add Command
+
+Adds the provided package to your feed in the following layout instead of making a simple copy of the nupkg file. For this command, only feeds which are folders or UNC shares are allowed. In order to expand all the files in the package to your feed, use the -Expand switch. The following layout has significant performance benefits, when performing a restore or an update against your feed, compared to a folder of nupkg files.
```

Comment:
Q: Should we use the term `feed` here? Perhaps use the more generic term `package source` instead?

Feed might be confusing, especially in the scenario where only file system folders or UNC shares are allowed, where there is no actual `feed`.